common/src/main/java/com/aliyun/rss/common/network/protocol/PushData.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.aliyun.rss.common.network.protocol;
+
+import com.google.common.base.Objects;
+import io.netty.buffer.ByteBuf;
+
+import com.aliyun.rss.common.network.buffer.ManagedBuffer;
+import com.aliyun.rss.common.network.buffer.NettyManagedBuffer;
+
```

Comment:
Except for this class, we'd better add annotations to all public classes for better understanding and code maintenance. Otherwise, after adding the check style file, the compile process may fail.

Reply:
These important data structures or APIs, such as pushing data or fetching data,  requires more careful thinking. It is better to abstract it into an interface, and at least consider how to adapt to Spark and Flink.